src/universal/Atmosphere.js

```diff
@@ -71,25 +138,16 @@ export default class Atmosphere extends Environment {
     });
     const resJson = await res.json();
     const {errors} = resJson;
-    if (!errors) return resJson;
-    const errorObj = makeErrorObj(errors);
-    return Promise.reject(errorObj);
+    if (errors) {
+      throw new Error(errors[0].message);
```

Comment:
Does the relay runtime catch these errors? Let's make sure *all* the graphql errors bubble up to the client, not just `errors[0]`.